Benchmarks/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using BenchmarkDotNet.Running;
+using System;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Characteristics;
+using System.IO;
+
+namespace Benchmarks
+{
+    public class Program
+    {
+        static void Main(string[] args) => BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args, new CustomConfig().Config);
+
+        private class CustomConfig : Attribute, IConfigSource
+        {
+            public IConfig Config { get; } = DefaultConfig.Instance;
+
+            public CustomConfig()
+            {
+                // Test CsWinRT projection
+                var job = Job.Default
+                    .WithPlatform(BenchmarkDotNet.Environments.Platform.X64)
+                    .WithArguments(
+                        new Argument[] {
+                            new MsBuildArgument("/p:platform=x64"),
+                            new MsBuildArgument("/p:MsAppxPackageTargets=%temp%\\EmptyMsAppxPackage.Targets")

```

Comment:
This isn't needed any more - I removed it from the pipeline and build.cmd (so you won't have a %temp%\EmptyMsAppxPackage.Targets anyway)

Reply:
Yea it doesn't seem to be needed.  Removed it.